<ui version="4.0" stdsetdef="1" >
  <class>PhraseBookBox</class>
  <comment>*********************************************************************
**
** Copyright (C) 1992-$THISYEAR$ $TROLLTECH$. All rights reserved.
**
** This file is part of the $MODULE$ of the Qt Toolkit.
**
** $TROLLTECH_DUAL_LICENSE$
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
  <widget class="QDialog" name="PhraseBookBox" >
    <property name="objectName" >
      <string notr="true" >phrase book box</string>
    </property>
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>604</width>
        <height>480</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Edit Phrase Book</string>
    </property>
    <property name="whatsThis" stdset="0" >
      <string>This window allows you to add, modify, or delete phrases in a phrase book.</string>
    </property>
    <layout class="QHBoxLayout" >
      <property name="objectName" >
        <string notr="true" >unnamed</string>
      </property>
      <item>
        <layout class="QVBoxLayout" >
          <property name="objectName" >
            <string notr="true" >unnamed</string>
          </property>
          <item>
            <layout class="QGridLayout" >
              <property name="objectName" >
                <string notr="true" >unnamed</string>
              </property>
              <item row="1" column="0" >
                <widget class="QLabel" name="target" >
                  <property name="objectName" >
                    <string notr="true" >target</string>
                  </property>
                  <property name="text" >
                    <string>&amp;Translation:</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1" >
                <widget class="QLineEdit" name="targetLed" >
                  <property name="objectName" >
                    <string notr="true" >targetLed</string>
                  </property>
                  <property name="whatsThis" stdset="0" >
                    <string>This is the phrase in the target language corresponding to the source phrase.</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="0" >
                <widget class="QLabel" name="source" >
                  <property name="objectName" >
                    <string notr="true" >source</string>
                  </property>
                  <property name="text" >
                    <string>S&amp;ource phrase:</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="1" >
                <widget class="QLineEdit" name="definitionLed" >
                  <property name="objectName" >
                    <string notr="true" >definitionLed</string>
                  </property>
                  <property name="whatsThis" stdset="0" >
                    <string>This is a definition for the source phrase.</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1" >
                <widget class="QLineEdit" name="sourceLed" >
                  <property name="objectName" >
                    <string notr="true" >sourceLed</string>
                  </property>
                  <property name="whatsThis" stdset="0" >
                    <string>This is the phrase in the source language.</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="0" >
                <widget class="QLabel" name="definition" >
                  <property name="objectName" >
                    <string notr="true" >definition</string>
                  </property>
                  <property name="text" >
                    <string>&amp;Definition:</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <widget class="QTreeView" name="phraseList" >
              <property name="objectName" >
                <string notr="true" >phraseTable</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QVBoxLayout" >
          <property name="objectName" >
            <string notr="true" >unnamed</string>
          </property>
          <item>
            <widget class="QPushButton" name="newBut" >
              <property name="objectName" >
                <string notr="true" >newBut</string>
              </property>
              <property name="text" >
                <string>&amp;New Phrase</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Click here to add the phrase to the phrase book.</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="removeBut" >
              <property name="objectName" >
                <string notr="true" >removeBut</string>
              </property>
              <property name="text" >
                <string>&amp;Remove Phrase</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Click here to remove the phrase from the phrase book.</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="saveBut" >
              <property name="objectName" >
                <string notr="true" >saveBut</string>
              </property>
              <property name="text" >
                <string>&amp;Save</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Click here to save the changes made.</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="closeBut" >
              <property name="objectName" >
                <string notr="true" >closeBut</string>
              </property>
              <property name="text" >
                <string>Close</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Click here to close this window.</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer1" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>51</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Vertical</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <tabstops>
    <tabstop>sourceLed</tabstop>
    <tabstop>targetLed</tabstop>
    <tabstop>definitionLed</tabstop>
    <tabstop>phraseList</tabstop>
    <tabstop>newBut</tabstop>
    <tabstop>removeBut</tabstop>
    <tabstop>saveBut</tabstop>
    <tabstop>closeBut</tabstop>               
  </tabstops>
</ui>
